tests/components/building/parameter_client_test.py

```diff
@@ -82,7 +82,26 @@ def name() -> str:
     "critic_opt_state-network_agent_2",
 }
 
-expected_keys = expected_count_keys.union(expected_network_keys)
+normalisation_keys = {"norm_params"}
```

Comment:
again would be nice to replace this with a constant

Reply:
This is the main name for the normalisation parameters dict. Then we have two constant keys for this dictionary. 
```
obs_norm_key = constants.OBS_NORM_STATE_DICT_KEY
values_norm_key = constants.VALUES_NORM_STATE_DICT_KEY
```